addons/http_routing/views/http_routing_template.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <template id="http_error">
+    <template id="http_error" name="HTTP Error">
```

Comment:
Maybe just `Internal Error` or `Internal HTTP Error` would have been more user friendly, but we can start with this, it's already better than `http_error`